src/meta-srv/src/procedure/region_migration/update_metadata/upgrade_candidate_region.rs

```diff
@@ -144,6 +144,11 @@ impl UpdateMetadata {
         } = datanode_table_value.region_info.clone();
         let table_route_value = ctx.get_table_route_value().await?;
 
+        let region_distribution = region_distribution(&region_routes);
+        info!(
+            "Trying to update region routes to {region_distribution:?} for table: {}",
+            region_id.table_id()
```

Comment:
question: Is it recommended to mix two format styles?
```suggestion
            "Trying to update region routes to {:?} for table: {}",
            region_distribution, region_id.table_id()
```